src/cheri-pte-ext.adoc

```diff
@@ -16,9 +16,11 @@ NOTE: There is no explicit mechanism for enabling or disabling {cheri_pte_ext_na
 
 NOTE: A future version of this specification may include kernel revocation which may require an <<sstatusreg_pte,sstatus>>.SCRG bit.
 
-The remainder of this chapter specifies the behavior of PTE.CW, PTE.CRG and <<sstatusreg_pte,sstatus>>.UCRG.
+The remainder of this chapter jointly specifies the behavior of PTE.CW, PTE.CRG and <<sstatusreg_pte,sstatus>>.UCRG.
 
-If {cheri_pte_ext_name} is _not_ implemented then PTE.CRG and <<sstatusreg_pte,sstatus>>.UCRG are considered to exist but to be read-only-zero for the purpose of the specification, however a CHERI-aware hart running a VM-enabled OS is strongly recommended to support {cheri_pte_ext_name}.
+NOTE: The description below assumes that {cheri_pte_ext_name} has been implemented.
+ If that is _not_ the case then PTE.CRG and <<sstatusreg_pte,sstatus>>.UCRG should be taken as read-only-zero for purpose of the description in the remainder of this chapter only.
```

Comment:
I think this is ok now in that what it says is what we want to be true, but it would be better to split the description up into first CW on its own and then in some Svucrg[^1] what CRG adds on top, which is then text you can just not read if you don't implement that. But if you want to defer that to at least get the spec correct first that's ok.

[^1]: A more RISC-V-y name for what is currently called Zcheripte, which is definitely not a correct name for the thing given it (a) is a privileged spec extension (so should be Sfoo) (b) pertains to virtual memory (so should be Svfoo)

Reply:
I agree that it would be better to split the descriptions but I can't do that in the near term, so I think this will have to do for now.

Reply:
resolving for now - I appreciate that the structure should be improved